os/arch/arm/src/amebad/amebad_flash.c

```diff
@@ -350,6 +350,9 @@ FAR struct mtd_dev_s *up_flashinitialize(void)
 		priv->mtd.write = amebad_write;
 #endif
 		priv->nsectors = AMEBAD_NSECTORS;
+		u8 chip_id[4];
+		flash_read_id(NULL, chip_id, 4);
+		lldbg("Manufacturer : %d memory type : %d capacity : %d\n", chip_id[0], chip_id[1], chip_id[2], chip_id[3]);

```

Comment:
`chip_id` is unsigned type, so I think it is better to print with `%u` instead of `%d`.

Reply:
I changed code with your advice
Thank you :)